contents/news/2023-05-security.md

```diff
@@ -11,10 +11,10 @@ illustration:
 
 # Training to cybersecurity
 
-In an ever-evolving digital world, cybersecurity is a major concern for businesses. The DiagRAMS policy is now enriched with a new training program provided by the french national cybersecurity agency.
+In an ever-evolving digital world, cybersecurity is a major concern for businesses. The DiagRAMS policy is now enriched with a new training program provided by the French national cybersecurity agency.
 
 We are proud to announce our team recently obtained the cybersecurity certification issued by the ANSSI cybersecurity academy.
 
-We think that despite developpers takes a major role in sceuring our platform, any DiagRAMS collaborator should have the basic skill set to avoid well known security issues.
+We think that despite developers take a major role in securing our platform, any DiagRAMS collaborator should have the basic skill set to avoid well known security issues.
```

Comment:
```suggestion
We think that despite developers taking a major role in securing our platform, any DiagRAMS collaborator should have the basic skill set to avoid well known security issues.
```

Reply:
done